vendor/manifest

```diff
@@ -248,10 +248,10 @@
 		},
 		{
 			"importpath": "github.com/davecgh/go-xdr/xdr2",
-			"repository": "https://github.com/davecgh/go-xdr",
+			"repository": "https://github.com/rasky/go-xdr",
```

Comment:
yes, meant to mention that - this pulls in davecgh/go-xdr#6 until it lands, we need it for xdr optional pointer field support.